src/backend/browserview.ts

```diff
@@ -0,0 +1,146 @@
+import { BrowserView, BrowserWindow, ipcMain } from 'electron/main'
+import {
+  IBrowserView,
+  IBrowserViewIdentifier,
+  IBrowserViewOptions
+} from 'common/types/browserview'
+import { getMainWindow } from 'backend/main_window'
+
+class NodeBrowserView extends IBrowserView {
+  private view: BrowserView
+  initialURL: string
+  URLchanged: { (): void }[] = []
+
+  constructor(
+    identifier: IBrowserViewIdentifier,
+    { initialURL }: IBrowserViewOptions
+  ) {
+    super()
+    this.view = new BrowserView()
+    this.initialURL = initialURL
+    this.view.webContents.on('did-navigate-in-page', () => {
```

Comment:
Not completely sure about this approach. It has many issues. 
1. Creates two different URLchanged variables, for backend and for frontend
2. RemoteBrowserView is intended to be a mere reflection of NodeBrowserView, so you can create many instances safely. In this case it will not work because the callbacks are in the frontend.
3. However putting them on backend isn't very useful, because you can't send methods via IPC.
@flavioislima